config/confighttp/confighttp.go

```diff
@@ -400,3 +422,39 @@ func maxRequestBodySizeInterceptor(next http.Handler, maxRecvSize int64) http.Ha
 		next.ServeHTTP(w, r)
 	})
 }
+
+func denyIPRanges(next http.Handler, ipRanges []*net.IPNet) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
We should not do this because this is executed for every request and adds unnecessary overhead. You better change the [Server](https://pkg.go.dev/net/http#Server) `ConnState` function and on the `StateNew` or wrap the TCPListener and do that in the "Accept" wrapper.

Reply:
Done!